NEWS.md

```diff
@@ -1,6 +1,14 @@
 
 # tidyselect (development)
 
+* `starts_with()`, `ends_with()`, `contains()`, and `matches()` now
```

Comment:
Maybe make it a little more clear that it’s always OR? (In the text, not just the code)